src/components/Docs/SnippetViewer/BatchCheckRequestViewer.tsx

```diff
@@ -26,28 +26,327 @@ function batchCheckRequestViewer(lang: SupportedLanguage, opts: BatchCheckReques
+        ? `
+    ContextualTuples: []ClientTupleKey{
+      ${check.contextualTuples
+        .map(
+          (tuple) => `
+        {
+          User: "${tuple.user}",
+          Relation: "${tuple.relation}",
+          Object: "${tuple.object}",
+        },`,
+        )
+        .join('')}
+    },`
+        : ''
+    }
+  },`,
+        )
+        .join('')}
+}
+options := ClientBatchCheckOptions{${modelId ? `\n  AuthorizationModelId: PtrString("${modelId}"),\n` : ''}}
+data, err := fgaClient.BatchCheck(context.Background()).Body(requestBody).Options(options).Execute()
+/*
+data = [${checks
+        .map(
+          (check) => `{
+  Allowed: ${check.allowed},
+  Request: {
+    User: '${check.user}',
+    Relation: '${check.relation}',
+    Object: '${check.object}'${
+      check.contextualTuples
+        ? `,\n    ContextualTuples: [${check.contextualTuples
+            .map(
+              (tuple) => `{
+      User: '${tuple.user}',
+      Relation: '${tuple.relation}',
+      Object: '${tuple.object}'
+    }`,
+            )
+            .join(',\n    ')}]
+  `
+        : '\n  '
+    }}
+}`,
+        )
+        .join(', ')}]
+*/
+`;
 
     case SupportedLanguage.DOTNET_SDK:
-      return '';
+      return `// The .NET SDK does not yet support server-side batch checks. This currently just calls the check endpoint in parallel.
+
+var body = new ClientBatchCheckRequest {
+  Checks = new List<ClientCheckRequest>() {
+    ${checks
+      .map(
+        (check) => `new() {
+      User = "${check.user}",
+      Relation = "${check.relation}",
+      Object = "${check.object}",${
+        check.contextualTuples
+          ? `,
+      ContextualTuples = new List<ClientTupleKey>() {
+        ${check.contextualTuples
+          .map(
+            (tuple) => `new() {
+          User = "${tuple.user}",
+          Relation = "${tuple.relation}",
+          Object = "${tuple.object}",
+        }`,
+          )
+          .join(',')},
+      },`
+          : ''
+      }
+    },`,
+      )
+      .join('\n    ')}
+  },
+}
+var options = new ClientBatchCheckOptions {${modelId ? `\n  AuthorizationModelId: "${modelId}",\n` : ''}}
+var response = await fgaClient.BatchCheck(body, options);
+/*
+response.Responses = [${checks
+        .map(
+          (check) => `{
+  Allowed: ${check.allowed},
+  Request: {
+    User: '${check.user}',
+    Relation: '${check.relation}',
+    Object: '${check.object}'${
+      check.contextualTuples
+        ? `,\n    ContextualTuples: [${check.contextualTuples
+            .map(
+              (tuple) => `{
+      User: '${tuple.user}',
+      Relation: '${tuple.relation}',
+      Object: '${tuple.object}'
+    }`,
+            )
+            .join(',\n    ')}]
+  `
+        : '\n  '
+    }}
+}`,
+        )
+        .join(', ')}]
+*/
+`;
 
     case SupportedLanguage.PYTHON_SDK:
-      return '';
+      return `# Requires >=v0.9.0 for the server side BatchCheck, otherwise will be calling the check endpoint in parallel.
```

Comment:
Similar to the JS example, I don't think it's compatible on the arguments so I'm not sure this is correct